subsys/esb/esb.c

```diff
@@ -2049,6 +2049,15 @@ int esb_flush_tx(void)
 	tx_fifo.back = 0;
 	tx_fifo.front = 0;
 
+	for (size_t i = 0; i < CONFIG_ESB_TX_FIFO_SIZE; i++) {
+		ack_pl_wrap[i].in_use = false;
+		ack_pl_wrap[i].p_next = 0;
```

Comment:
Shouldn't we use `NULL` for null pointers?
I see that everywhere in this file is `0` for null pointers, that's something that we may consider in the future, but for now the PR look good to me.

Reply:
You're right. Updated.